src/OpenApi/SwaggerSpecs.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Picqer\BolRetailerV8\OpenApi;
+
+class SwaggerSpecs
+{
+    private $specs = [];
+
+    public function __construct($specs = [])
+    {
+        $this->specs = $specs;
+    }
+
+    public function load(string $file): SwaggerSpecs
+    {
+        $content = file_get_contents($file);
+        $content = $this->replaceErroneousCharacters($content);
+
+        $this->specs = json_decode($content, true);
+
+        return $this;
+    }
+
+    private function replaceErroneousCharacters(string $content): string
+    {
+        $replacements = [
+            ' ' => ' ', // 'ENSP' space
```

Comment:
Dit ziet er een beetje vreemd uit. Misschien goed om, als de keys lijken op de replacements, een hexcode te gebruiken? Ik weet niet precies welke het is, maar dit voelt fout gevoelig voor mij.

Reply:
Hmm, in mijn editor ziet dat er anders uit. Je voorstel is denk ik wel handig ja.

Reply:
Ah ja, ik heb het niet in PHPStorm geopend, maar in GitHub _lijkt_ het exact hetzelfde 😄